docs/events/security_alerts/symbols_loaded.md

```diff
@@ -24,8 +24,8 @@ starts with the prefix will be whitelisted.
 The use is only with the `!=` operator, and wildcards aren't supported.
 
 ## Arguments
-* `library_path`:`const char*`[K] - the path of the file written.
-* `symbols`:`const char*const*`[U,TOCTOU] - the first 20 bytes of the file.
+* `library_path`:`const char*`[K] - the path of the file shared object loaded.
```

Comment:
```suggestion
* `library_path`:`const char*`[K] - path of whitelisted shared object files.
```

I think that would be more accurate?

Reply:
Hmm not quite because it is given that only shared objects which are not whitelisted can invoke the event.
If it was a blacklist it we could probably write it in the description, but whitelist is clumsy.

However, it should probably be:
```suggestion
* `library_path`:`const char*`[K] - path of the shared object file loaded.
```

Reply:
Right right, confused it with documentation for the filters. 
But yes what you suggested would be better.